notification-service/src/recievers.ts

```diff
@@ -0,0 +1,33 @@
+import { RECEIVERS } from './variables';
+
+const validateReceivers = function (receivers: string[]) {
+    const invalidEmails: string[] = [];
+    receivers.forEach(receiver => {
+        if (!receiver.includes('@') || !receiver.includes('.')) {
```

Comment:
`.@` would be a valid email, right? What about using a package instead?
- https://www.npmjs.com/package/email-validator
- https://www.npmjs.com/package/validator